manifests/broker.pp

```diff
@@ -15,7 +15,7 @@
     Service[$broker_service] -> Exec['migrate_pulp_db']
   } else {
     if $pulp::messaging_transport == 'qpid' {
-      package { 'qpid-tools': ensure => installed } -> Class['pulp::service']
+      Class['qpid::tools'] -> Class['pulp::service']
```

Comment:
You gave the example though of qpid being external to the pulp - does it need qpid-tools still on the pulp in that case?

If so, you can `include qpid::tools` here too